build.gradle

```diff
@@ -1409,6 +1409,7 @@ project(':storage') {
   dependencies {
     implementation project(':storage:api')
     implementation project(':clients')
+    implementation project(':metadata')
```

Comment:
I will remove this dependency once I move `ApiMessageAndVersion` to `clients` module. This is done in a followup PR.